whylog/assistant/regex_assistant/guessing.py

```diff
@@ -0,0 +1,39 @@
+import re
+
+from whylog.assistant.pattern_object import ParamGroup, PatternObject
+from whylog.assistant.regex_assistant.regex import (
+    create_obvious_regex, regex_from_group_spans, verify_regex
+)
+from whylog.assistant.spans_finding import find_date_spans
+
+
+def guess_regex_objects(text):
```

Comment:
This name is not clear. What does mean guessing regex object? What is regex_object? What is the difference beetwen regex object and regex from re library? Why this function returns list of two objects? Why you don't use touple?